ament_cmake_cppcheck/cmake/ament_cmake_cppcheck_lint_hook.cmake

```diff
@@ -33,7 +33,14 @@ if(_source_files)
     list(APPEND _all_include_dirs ${ament_cmake_cppcheck_ADDITIONAL_INCLUDE_DIRS})
   endif()
 
-  # Get exclde paths for added targets
+  # Language
+  set(_language "")
+  if(DEFINED ament_cmake_cppcheck_LANGUAGE)
+    set(_language LANGUAGE ${ament_cmake_cppcheck_LANGUAGE})
+    message(STATUS "Configured cppcheck language : ${_language}")
```

Comment:
The message should only print `ament_cmake_cppcheck_LANGUAGE` - not `_language`.

Nitpick: remove the space before the colon.